internal/ingress/controller/controller.go

```diff
@@ -188,7 +188,9 @@ func (n *NGINXController) syncIngress(interface{}) error {
 				// it takes time for Nginx to start listening on the port
 				time.Sleep(1 * time.Second)
 			}
-			err := n.ConfigureDynamically(&pcfg)
+
+			url := fmt.Sprintf("http://localhost:%d/configuration/backends", n.cfg.ListenPorts.Status)
+			err := configureDynamically(&pcfg, url)
```

Comment:
I realize it makes testing either but to me this feels counterintuitive for `configureDynamically` to not know about the endpoint URL itself and get it from a caller.